libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/TriggerConditions/Microsoft.OnClearProperty.schema

```diff
@@ -0,0 +1,21 @@
+{
+    "$schema": "https://raw.githubusercontent.com/microsoft/botbuilder-dotnet/master/schemas/component.schema",
+    "$role": "union(Microsoft.ITriggerCondition)",
+    "title": "ClearProperty Event",
+    "description": "This defines the actions to take when a ClearProperty event is raised",
```

Comment:
Can you update the language around this to match the other Onxxx definitions?
    "title": "On clear property",
    "description": "Actions to perform when specified property needs to cleared.",